docs/ug/configFiles.md

```diff
@@ -40,11 +40,18 @@ Given below are the details of the various config files used by RepoSense.
 | Ignore Commits List<sup>*+</sup> | The list of commits to ignore during analysis. For accurate results, the commits should be provided with their full hash. Additionally, a range of commits can be specified using the `..` notation e.g. `abc123..def456` (both inclusive). |
```

Comment:
The `ug` folder also has its own copy of `repo-config.csv`. Remember to add the new column headers and commas